student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
```

Comment:
Kindly include `classroom` as an argument of the `initialize` method, otherwise the value of `classroom` in `@classroom = classroom` will always be undefined.

Reply:
Hi @KDlamini,